Header header
string task
string reference_topic
string reference_frame
time reference_time
float32 quality
AnnotationBox[] boxes
AnnotationPolygon[] polygons
